### PR TITLE
Support audio duration mismatch tolerance in `MixedCut.load_audio()`

### PR DESCRIPTION
**Pull request by pzelasko**

### CODE REVIEW

File: lhotse/workflows/meeting_simulation/speaker_independent.py [hunk 1]
```diff
@@ -218,3 +172,55 @@ def _simulate_worker(utterances: CutSet) -> MixedCut:
...
+        loc + npr.exponential(scale=scale, size=len(utterances[i]))
+        for i in range(len(utterances))
+    ]
+
+    # Create the meeting.
+    mixture = _create_mixture(utterances, silence_durations)
+    return mixture
+
+
+def _create_mixture(
```
Comment (desh2608) [inline lhotse/workflows/meeting_simulation/speaker_independent.py:204]: Is there a reason why this function is moved outside the class? The algorithm is specific to this particular simulator.

Comment (pzelasko) [reply lhotse/workflows/meeting_simulation/speaker_independent.py]: Makes sense, I changed this back and used the same "trick" as in the simulator.

Comment (desh2608) [reply lhotse/workflows/meeting_simulation/speaker_independent.py]: Thanks. Could you do the same for `_simulate_worker` (i.e. make it a function of the specific class)? The logic for those is also tied to that algorithm.

Comment (pzelasko) [reply lhotse/workflows/meeting_simulation/speaker_independent.py]: That would complicate the workarounds for parallelization; anyway each simulation method has a separate file, so I think it should be OK and actually consistent with other functions in lhotse that are being parallelized this way, e.g. `_X_single()` functions at the end of `lhotse/cut/set.py`.

Comment (desh2608) [reply lhotse/workflows/meeting_simulation/speaker_independent.py]: Okay, LGTM.

**Review by desh2608 — APPROVED** [review]

**Pull request merged**